src/import/import-generator.c

```diff
@@ -257,8 +257,12 @@ static int transfer_generate(const Transfer *t, size_t c) {
 
         assert(t);
 
+        _cleanup_free_ char *e = unit_name_escape(t->remote);
```

Comment:
`remote` may be too long for unit name. I think it is better to use `local` as previous.

At least, please add assertion for `t->remote`, or `t->local`.

Reply:
I reworked this now, to use the full path of the image we generate. that's a lot more useful, since it means we can use it sanely for dependencies.